tornado/log.py

```diff
@@ -57,6 +64,8 @@ def _stderr_supports_color():
                 color = True
         except Exception:
             pass
+    elif colorama and os.isatty(sys.stderr.fileno()):
```

Comment:
Originally this was due to a misunderstanding on my part of #920. Using `os.isatty` means we don't need to additionally check if `sys.stderr` has the `isatty` attribute. From [`io.IOBase`](https://docs.python.org/3/library/io.html#io.IOBase), `sys.stderr` should always have a `fileno` attribute, however.

I didn't want to change the line that was already working and have something break, but it would probably make sense to just check once if `os.isatty(...)` or `sys.stderr.isatty()`. I'll rework that.